Formula/n/netatalk.rb

```diff
@@ -0,0 +1,81 @@
+class Netatalk < Formula
+  desc "File server for Macs, compliant with Apple Filing Protocol (AFP)"
+  homepage "https://netatalk.io"
+  url "https://github.com/Netatalk/netatalk/releases/download/netatalk-4-0-8/netatalk-4.0.8.tar.xz"
+  sha256 "d09d591521b27b91b1c2a6255a2e059af8c9fda43570f983f0a145706e5f7628"
+  license all_of: [
+    "GPL-2.0-only",
+    "GPL-2.0-or-later",
+    "LGPL-2.0-only",
+    "LGPL-2.1-or-later",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "MIT",
+  ]
+  head "https://github.com/Netatalk/netatalk.git", branch: "main"
+
+  depends_on "docbook-xsl" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+
+  depends_on "berkeley-db@5" # macOS bdb library lacks DBC type etc.
+  depends_on "cracklib"
+  depends_on "libevent"
+  depends_on "libgcrypt"
+  depends_on "mysql"
+  depends_on "openldap" # macOS LDAP.Framework is not fork safe
+
+  uses_from_macos "libxslt" => :build
+
+  uses_from_macos "krb5"
+  uses_from_macos "libxcrypt"
+  uses_from_macos "perl"
+
+  on_linux do
+    depends_on "avahi" # on macOS we use native mDNS instead
+    depends_on "cups" # used by the AppleTalk print server
+    depends_on "libtirpc" # on macOS we use native RPC instead
+  end
+
+  patch do
+    url "https://github.com/Netatalk/netatalk/commit/206fb7771862b9b98452c934dac884aaa397c8ca.patch?full_index=1"
+    sha256 "fd448734556daf0344be4fa0bb09e4704c4123078ad804069d288aa0e3e104d6"
+  end
+
+  def install
+    inreplace "distrib/initscripts/macos.netatalk.in", "@sbindir@", opt_sbin
+    inreplace "distrib/initscripts/macos.netatalk.plist.in", "@sbindir@", opt_sbin
```

Comment:
There's still a `@bindir@` in the PATH portion of macos.netatalk.plist.in, by the way.

Reply:
You're right. My mistake. Fix pushed!